### PR TITLE
Correct Usage In README.md

### PR DESCRIPTION
**Pull request by mearns**
It doesn't return the path, it returns a Promise.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -15,9 +15,15 @@ $ npm install tempdir --save
 ```javascript
 const tempdir = require('tempdir');
 
-let path = tempdir();
+tempdir().then(
```
Comment (vadimdemedes) [inline Readme.md]: I think

``` js
tempdir().then(path => {
  // e.g. path = /var/folders/y6/dn6f5hk50qq1167fvxv6vwrw0000gp/T/4d95a74b-9676-447d-ae08-fdeb6e281609
});
```

would be prettier.

Comment (mearns) [inline Readme.md]: Sure, I mean it's your repo, you can do what you want with it =). Personally I think it's worth pointing out how to handle errors, but I suppose anyone familiar with promises should already know that.

**Comment by mearns** [comment]
Updated Readme in latest commit, per @vdemedes' comment.

**Comment by vadimdemedes** [comment]
Thanks!

**Pull request merged**